azkaban-web-server/src/main/java/azkaban/imagemgmt/daos/ImageVersionDao.java

```diff
@@ -0,0 +1,11 @@
+package azkaban.imagemgmt.daos;
```

Comment:
Can you please add open source disclaimer for all the new classes above package?

Reply:
Thanks for reminding this. Done.